qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -2,7 +2,7 @@ Feature: Visit types Management
 
   Background:
     Given a user clicks on the system administration link from the home page
-    Then the system loads the administration page
+    Then the administration page gets loaded
     And a user clicks on the manage Visit Types link
```

Comment:
What happens when the user clicks on the manage visit types link?

Dont you feel you should have something like `Then the manage visit types page is loaded` and you can have a function in the steps definition file e.g
```
public void systemLoardsManageVisitTypexxxxx() {
assertTrue(textExists("any text that appears when the manage visit type page loads"));
}
```